packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -421,10 +421,6 @@ export default {
             await Promise.all(promises);
             this.resetLoadingState();
 
-            if (this.shouldLoadCustomerNameFromClaims) {
```

Comment:
I don't think we need the `shouldLoadCustomerNameFromClaims` computed property any more?